.env.production

```diff
@@ -0,0 +1,9 @@
+# The port for the development server
+PORT=8080
+
+VUE_APP_WIKIBASE_API_URL='https://www.wikidata.org/w/api.php'
+VUE_APP_QUERY_SERVICE_EMBED_URL='https://query.wikidata.org/embed.html#'
```

Comment:
This should not have the trailing `#`. That is conceptually part of the hash that we appending in code